README.md

```diff
@@ -70,4 +70,49 @@ To make use of a built image, run:
         docker pull docker.montagu.dide.ic.ac.uk:5000/montagu-reporting-api:master
         docker run --rm -p 8080:8080 -v {PATH_TO_ORDERLY}:/orderly docker.montagu.dide.ic.ac.uk:5000/montagu-reporting-api:master
 
-replacing `{PATH_TO_OPRDERLY}` with an absolute path to an Orderly directory.
+replacing `{PATH_TO_ORDERLY}` with an absolute path to an Orderly directory.
+
+
+## Orderly Web database schema
+
+Tables required by Orderly Web, relating to users and permissions, are added to the Orderly database by this application. Code and migrations for 
+this can be found in the ```migrations``` folder.  Migrations are run by the docker container defined by 
+```migrations/Dockerfile``` which uses [Flyway](https://flywaydb.org/) to apply migrations defined in 
+```migrations/sql``` using Flyway configiration defined in ```migrations/flyway.conf```.
+The ```scripts/teamcity-migrate.sh``` script tests, builds and pushes this docker image as part of the Team City build, 
+and migrations can also be tested locally with ```scripts/local-test-migrate.sh``` 
+
+We don't create a schema as such in the Orderly database, as Sqlite does not support schema. Instead we prefix all our 
+tables' names with "orderlyweb".
+
+These tables are:
+### orderlyweb_user
+The users of OrderlyWeb, however they are authenticated.
```

Comment:
Shall we make an executive decision between "Orderly Web" (in titles and line 78) and "OrderlyWeb" here? I think I prefer "OrderlyWeb"!